config/sudoers

```diff
@@ -24,12 +24,16 @@ vx-admin ALL=(root:ALL) NOPASSWD: /vx/admin/admin-functions/set-clock.sh
 vx-admin ALL=(root:ALL) NOPASSWD: /vx/admin/admin-functions/lockdown.sh
 vx-admin ALL=(root:ALL) NOPASSWD: /vx/admin/admin-functions/generate-key.sh
 vx-admin ALL=(root:ALL) NOPASSWD: /vx/admin/admin-functions/setup-boot-entry.sh
+vx-admin ALL=(root:ALL) NOPASSWD: /usr/local/bin/tpm2-totp
+vx-admin ALL=(root:ALL) NOPASSWD: /vx/code/app-scripts/mount.sh
+vx-admin ALL=(root:ALL) NOPASSWD: /vx/code/app-scripts/umount.sh
 vx-ui ALL=(root:ALL) NOPASSWD: /vx/code/config/admin-functions/timedatectl
 vx-ui ALL=(root:ALL) NOPASSWD: /usr/bin/brightnessctl
 vx-ui ALL=(root:ALL) NOPASSWD: /usr/local/bin/tpm2-totp
-vx-ui ALL=(root:ALL) NOPASSWD: /vx/ui/ui-functions/sign.sh
+vx-ui ALL=(root:ALL) NOPASSWD: /vx/code/app-scripts/sign.sh
+vx-ui ALL=(root:ALL) NOPASSWD: /vx/code/app-scripts/mount.sh
+vx-ui ALL=(root:ALL) NOPASSWD: /vx/code/app-scripts/umount.sh
```

Comment:
I think that we could grant `sudo` access either to the original or the symlink and both would work (both already exist in this file) but for simplicity I think granting access to the source is better because then it is the same across users, we could grant perms in one statement if multiple users need the same perms, etc.